lib/axios.ts

```diff
@@ -28,8 +28,17 @@ export default async function(step: any, config) {
 }
 */
 
-export default async function(step: any, config) {
+export default async function(step: any, config, sign?) {
   // XXX warn about mutating config object... or clone?
+  if (sign) {
+    const payload = {
+      request_data: config,
+      sign,
+    }
+    const oauthEndpoint = "enlb0ktwajm8sen"
+    const oauthSignature = (await axios.post(`https://${oauthEndpoint}.m.pipedream.net?pipedream_response=1`, payload)).data
+    config.headers = oauthSignature
```

Comment:
```
if (!config.headers) config.headers = {}
config.headers.Authorization = oauthSignature.Authorization // check case of key in returned oauthSignature object
```

... or update the workflow to just return the Authorization header value (instead of the object). But still, just update `Authorization` header because the config may have other header values.